packages/react/src/components/linkbox/LinkboxWithImage.stories.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import { Linkbox } from './Linkbox';
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+// @ts-ignore
+import exampleImage from './static/example-img-small.png';
```

Comment:
If possible, it would be better if we used higher resolution example images here. These get pixelated quite easily.

Higher resolution (@2x, @3x) images can be found in the documentation folder static assets.
![image](https://user-images.githubusercontent.com/4214671/130607319-56f290a1-75bb-42d4-ab7f-2f7b67bd482f.png)

Reply:
- [ ] Use higher resolution images